### PR TITLE
Ignore files in train folder

### PR DESCRIPTION
**Pull request by Ornias1993**
Currently a train folder can only contain folder and no files.

However: it would be nice ignoring files in the train folder completely, because they are totally irrelevant.

### CODE REVIEW

File: catalog_validation/validation.py [hunk 1]
```diff
@@ -36,8 +36,11 @@ def validate_train(train_path):
     verrors.check()
 
     for catalog_item in os.listdir(train_path):
+        catalog_item_path = os.path.join(train_path, catalog_item)
+        if catalog_item.startswith('.') or not os.path.isdir(catalog_item_path):
```
Comment (sonicaj) [inline catalog_validation/validation.py:40]: Out of curiosity, do we need/require directories starting with `.` under a train directory ?

Comment (sonicaj) [reply catalog_validation/validation.py:40]: ping @Ornias1993

**Comment by sonicaj** [comment]
Closing this based on our discussion in slack

**Comment by Ornias1993** [comment]
@sonicaj Thank you! :)

**Pull request closed without merging**